packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -163,6 +163,45 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
     cancelAnimationFrame(this._reqID);
   }
 
+  public shouldComponentUpdate(nextProps: Readonly<IModifiedCartesianChartProps>): boolean {
+    let isUpdate: boolean = false;
```

Comment:
Is update is not needed in product code.
It can be there only in example code.

Reply:
done